whatshap/cli/stats.py

```diff
@@ -27,8 +27,8 @@ def add_arguments(parser):
         "and ignore all other variants.")
     add("--block-list", metavar="FILE", help="Write list of all blocks to FILE (one block per line)")
     add("--chromosome", dest="chromosomes", metavar="CHROMOSOME", default=[], action="append",
-        help="Name of chromosome to process. If not given, all chromosomes in the "
-        "input VCF are considered. Can be used multiple times")
+        help="Name of chromosome(s) to process. If not given, all chromosomes in the "
+        "input VCF are considered. Accepts multiple inputs or a comma-separated list.")
```

Comment:
"Accepts multiple inpus" could also mean that you can write it like `--chromosome chr1 chr2 chr3` (argparse has support for this if I remember correctly). I think explicitly stating that the option can be used multiple times is less ambiguous.